odata-csdl/1 Introduction.md

```diff
@@ -11,7 +11,7 @@ This format is based on the OpenUI5 OData V4 Metadata JSON Format, see
 [OpenUI5](#_OpenUI5), with some extensions and
 modifications made necessary to fully cover OData CSDL Version 4.01.
 
-## ##subsec Changes from earlier Versions
+## ##subsec Changes from Earlier Versions
```

Comment:
@HeikoTheissen Why is this different from the OASIS template / starter document? Do you plan to change this also in the other specifications?